requirements-dev.txt

```diff
@@ -14,9 +14,9 @@ python-magic==0.4.18
 messytables==0.15.2
 progressbar==2.5
 
--e git+https://github.com/ckan/ckanext-dcat.git@v1.1.3#egg=ckanext-dcat
+-e git+https://github.com/ckan/ckanext-dcat.git@v1.2.0#egg=ckanext-dcat
```

Comment:
do we have this in our infra project  yet?

Reply:
It's coming, on the QOL-9055 branch.